demo/setup.py

```diff
@@ -1,14 +1,9 @@
 #/usr/bin/env python
 import io
 import re
-import sys
 
 from setuptools import setup, find_packages
 
-if sys.version_info[:3] < (3, 4):
-    raise SystemExit("Toga requires Python 3.4+.")
```

Comment:
Do we think this check is no longer necessary at all? Do the "python_requires" blocks below make this irrelevant?

Reply:
@phildini so I did some testing to be sure:

    $ pyenv local 3.4.8
    $ pipenv --python 3.4.8
    $ pipenv install -e src/core
    Obtaining file:///home/dan/Projects/toga/src/core
    Error:  An error occurred while installing -e src/core!
    toga-core requires Python '>=3.5' but the running Python is 3.4.8

Looks like python_requires alone is just what we need! :+1:

Reply:
🙌

Reply:
Confirmed - yes, this is all we need, and it's a much better way to handle it than a runtime check.